firefox-ios/Client/Extensions/UIView+Extension.swift

```diff
@@ -65,4 +65,11 @@ extension UIView {
         maskLayer.path = maskPath.cgPath
         layer.mask = maskLayer
     }
+
+    /// Getting a snapshot from a view using image renderer
+    var snapshot: UIImage {
+      UIGraphicsImageRenderer(size: bounds.size).image { _ in
+        drawHierarchy(in: bounds, afterScreenUpdates: true)
+      }
```

Comment:
I think this is 2 spaces indent instead of 4?
```suggestion
    var snapshot: UIImage {
        UIGraphicsImageRenderer(size: bounds.size).image { _ in
            drawHierarchy(in: bounds, afterScreenUpdates: true)
        }
```